repository.rosinstall

```diff
@@ -34,6 +34,11 @@
     version: F_fixing_kinetic_build
 
 - git:
```

Comment:
Please add comment that this dependency will not be needed when version x.y.x of ros_controllers is released.